<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>navigationWidget</class>
 <widget class="QWidget" name="navigationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QToolButton {
	width: 23px;
	height: 23px;
	color: #000000;
	border-radius: 3px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(228, 228, 228, 255), stop:1 rgba(201, 201, 201, 255));
}*/</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolButton" name="firstTBn">
     <property name="minimumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="toolTip">
      <string>首页</string>
     </property>
     <property name="text">
      <string>首页</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="prevTBn">
     <property name="minimumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="toolTip">
      <string>上一页</string>
     </property>
     <property name="text">
      <string>上一页</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBox">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> 页</string>
     </property>
     <property name="prefix">
      <string>第 </string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2000000000</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="nextTBn">
     <property name="minimumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="toolTip">
      <string>下一页</string>
     </property>
     <property name="text">
      <string>下一页</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="lastTBn">
     <property name="minimumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="toolTip">
      <string>尾页</string>
     </property>
     <property name="text">
      <string>尾页</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="showSpinBox">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> 条</string>
     </property>
     <property name="prefix">
      <string>显示 </string>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
